taplo.toml

```diff
@@ -0,0 +1,26 @@
+[formatting]
+# Align consecutive entries vertically.
+align_entries = false
+# Append trailing commas for multi-line arrays.
+array_trailing_comma = true
+# Expand arrays to multiple lines that exceed the maximum column width.
+array_auto_expand = true
+# Collapse arrays that don't exceed the maximum column width and don't contain comments.
+array_auto_collapse = false
+# Omit white space padding from single-line arrays
+compact_arrays = true
+# Omit white space padding from the start and end of inline tables.
+compact_inline_tables = false
```

Comment:
Just like with `compact_arrays`, I like to have spacing between `{` and `}`. If you agree in setting `compact_arrays = true`, we should change this to true as well:

```suggestion
# Include white space padding in inline tables for better readability.
compact_inline_tables = true
```

Reply:
that would have been the opposite outcome

`compact_inline_tables = false` results in:
```
async-std           = { version = "1.10.0", features = [ "attributes" ] }
```

`compact_inline_tables = true` results in:
```
async-std           = {version = "1.10.0", features = [ "attributes" ]}
```

we don't need to change anything here